assets/alathraextras/itemtypes/tool/bullseye/bullseyedisabledbow.json

```diff
@@ -0,0 +1,170 @@
+{
+	code: "bow",
+	variantgroups: [
+		{ code: "type", states: ["simple", "crude", "long", "recurve" ] },
+	],
+	class: "ItemBow",
+	maxstacksize: 1,
+	durabilityByType: {
+		"bow-crude": 180,
+		"bow-simple": 240,
+		"bow-long": 750,
+		"bow-recurve": 600
+	},
+	shape: {
+		base: "game:item/tool/bow/{type}",
+		alternates: [
+			{ base: "game:item/tool/bow/{type}-charge1" },
+			{ base: "game:item/tool/bow/{type}-charge2" },
+			{ base: "game:item/tool/bow/{type}-charge3" }
+		]
+	},
+	tool: "bow",
+	creativeinventory: { "general": ["*"], "items": ["*"], "tools": ["*"] },
+	attributes: {
+		heldItemPitchFollow: 1,
+		aimAnimationByType: {
+			"bow-crude": "bowaimcrude",
+			"bow-long": "bowaimlong",
+			"bow-recurve": "bowaimrecurve",
+			"*": "bowaim",
+		},
+		damageByType: {
+			"bow-crude": 3,
+			"bow-simple": 3.25,
+			"bow-long": 3.75,
+			"bow-recurve": 4
+		},
+		accuracyBonusByType: {
+			"bow-crude": -0.05,
+			"bow-simple": 0,
+			"bow-long": 0.2,
+			"bow-recurve": 0.3	
+		},
+		toolrackTransformByType: {
+			"bow-long": {
+				rotation: { x: 2, y: 5, z: 0 },
+				translation: { x: 0.2, y: 0.95 },
+				scale: 1.76
+			},
+			"*": {
+				rotation: { x: 2, y: 5, z: 0 },
+				translation: { y: 0.63 },
+				scale: 1.25
+			}
+		},
+			groundStorageTransformByType: {
+			"bow-simple":{
+				translation: { x: -0.4, y: -0.03, z: 0.315 },
+				rotation: { x: 26, y: -180, z: -90 },
+				scale: 0.88
+			},
+			"bow-crude":{
+				translation: { x: -0.4, y: -0.11, z: 0.345 },
+				rotation: { x: 26, y: -180, z: -90 }
+			},
+			"bow-long":{
+				translation: { x: -0.45, y: 0.1, z: 0.29 },
+				rotation: { x: 26, y: -180, z: -90 },
+				scale: 0.88
+			},
+			"*":{
+				translation: { x: -0.4, y: 0.1, z: 0.28 },
+				rotation: { x: 20, y: -180, z: -90 },
+				scale: 0.83
+			},
+		}
+	},
+	behaviors: [{ 
+		name: "GroundStorable",
+		properties: {
+			layout: 'WallHalves',
+			wallOffYByType: {
+				"bow-crude": 1,
+				"bow-simple": 1,
+				"*": 2,
+			},
+			sprintKey: true,
+			selectionBox: { x1: 0, y1: 0, z1: 0, x2: 1, y2: 0.1, z2: 1 },
+			collisionBox: { x1: 0, y1: 0, z1: 0, x2: 0, y2: 0, z2: 0 },
+		}
+	}],
+	fpHandTransformByType: {
+		"bow-recurve": {
+			translation: { x: 0, y: 0.5, z: 0 },
+			rotation: { x: 4, y: 1, z: 84 },
+			origin: { x: 0.5, y: 0, z: 0.5 },
+			scale: 2.42
+		},
+		"bow-long": {
+			translation: { x: 0, y: 0.5, z: 0 },
+			rotation: { x: 4, y: 1, z: 84 },
+			origin: { x: 0.5, y: 0, z: 0.5 },
+			scale: 3.41
+		},
+		"*": {
+			translation: { x: 0, y: 0.5, z: 0 },
+			rotation: { x: 4, y: 1, z: 84 },
```

Comment:
Let's use a standard for indentation that does not involve tabs. i.e. 2 tab or 4 tab indents

Reply:
@BinglPringl  most editors have a setting to choose indent for file. I think for vs this is what you need https://stackoverflow.com/questions/14167033/visual-studio-replace-tab-with-4-spaces